modules/samples/sample-clients/kafka-producer/src/main/java/org/wso2/sp/sample/kafka/client/KafkaClient.java

```diff
@@ -137,9 +134,10 @@ public static void main(String[] args) {
      * @param fileName     name of the file with events
      */
     private static String getMessageFilePath(String sampleNumber, String fileName) throws Exception {
+        String fileExtension = ".txt";
```

Comment:
Better if we can define fileExtension type as  constant in class. Any reason for moved this to method?

Reply:
Intellij IDEA suggested that this field can be moved to a local variable.

Reply:
Resolved in commit 72ef342